src/components/Editor/QuillEditor.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useRef, useState } from "react";
+import "../../css/quillEditor.css";
+import { useFirebase } from "react-redux-firebase";
+import { useDispatch, useSelector } from "react-redux";
+import { Prompt } from "react-router-dom";
+import { setCurrentStep } from "../../store/actions";
+import * as Y from "yjs";
```

Comment:
Can we think of a better name than `Y` ?

Reply:
Again It's directly taken from the documentation and it looks like its a de facto name in community. (like we use _ for lodash)